supabase/migrations/20260816120150_org_member_read_billing_default.sql

```
@@ -0,0 +1,499 @@
+INSERT INTO public.role_bindings (
+  principal_type,
+  principal_id,
+  role_id,
+  scope_type,
+  org_id,
+  granted_by,
+  reason,
+  is_direct
+)
+SELECT
+  public.rbac_principal_user(),
```

SQLFluff (4.2.2)

[error] Column expression without alias. Use explicit `AS` clause.

(AL03)